webapp-client/src/component/Navbar.js

```diff
@@ -0,0 +1,33 @@
+import logo from '../lobster-icon.jpg';
+import React, { Component } from 'react';
+
+class Navbar extends Component {
```

Comment:
For consistency purpose, because in other files we did "extends React.Component" without importing Component separately, maybe change this to React.Component?

Reply:
Done